src/Curators/DBALNewsArticleRepository.php

```diff
@@ -77,7 +64,14 @@ public function search(NewsArticleSearch $newsArticleSearch): NewsArticles
                 ->setParameter(':url', $newsArticleSearch->getUrl());
         }
 
+        if ($newsArticleSearch->getStartPage() > 1) {
+            $query = $query->setFirstResult(
```

Comment:
Is this zero-based? Should we add 1 if it's not?

Reply:
It is 1-based. So I added a check inside the mutator method https://github.com/cultuurnet/udb3-silex/pull/850/files#diff-16b9e8bcb01691eaad8afdce848124a1d0cae3fd8d5350f674a637dbdddb9f66R29

Reply:
I did understand the question wrong. The call `setFirstResult` is zero based but the pagination is one based.

For example: 2 items per page and page 3 is asked, this means that we need item (2 x 3) + 1 = 7. But because the DBAL offset is zero based we need item 6, so we can do 2 x 3.